.circleci/config.yml

```diff
@@ -1220,7 +1220,7 @@ workflows:
                 - php:7.2-fpm-alpine
                 - php:7.3-fpm-alpine
                 - php:7.4-fpm-alpine
-                - php:8.0.0RC4-fpm-alpine # Add RC5 when available
+                - php:8.0-rc-fpm-alpine
```

Comment:
`-rc-`?

Reply:
Yeah, they don't have a non-RC tag for GA yet, but that tag was the most recent one for PHP 8.0.